lib/ts/recipe/session/sessionClass.ts

```diff
@@ -162,4 +146,71 @@ export default class Session implements SessionContainerInterface {
             throw err;
         }
     };
+
+    async validateClaims(
+        claimValidators: SessionClaimValidator[],
+        userContext?: any
+    ): Promise<ClaimValidationError | undefined> {
```

Comment:
Let's move this function to somewhere where the user is not exposed to it (ideally inside the getSession function)